<ui version="4.0" >
 <class>BasicManageDialogClass</class>
 <widget class="QWidget" name="BasicManageDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BasicManageDialog</string>
  </property>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>生成统计数据</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>120</y>
     <width>411</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>80</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
